safe/utilities/test/test_clipper.py

```diff
@@ -666,6 +668,40 @@ def test_adjust_clip_extent(self):
             expected_adjusted_extent, adjusted_extent)
         self.assertEqual(adjusted_extent, expected_adjusted_extent, message)
 
+    def test_clip_vector_with_unicode(self):
+        """Test clipping vector layer with unicode attribute in feature.
+
+        This issue is described at Github #2262 and #2233
+        """
```

Comment:
Yes for now use None and add a TODO: This is a lousy hack, see above ticket, then add your above annotations to the ticket please.